bluebrain/repo-bluebrain/packages/touchdetector/package.py

```diff
@@ -118,3 +118,6 @@ def cmake_args(self):
                 )
 
         return args
+
+    def setup_build_environment(self, env):
+        env.prepend_path("CMAKE_PREFIX_PATH", self.spec["tbb"].package.component_prefix)
```

Comment:
What is the reason for this line? Naively, I'd expect Spack to automatically ensure packages can be found by CMake.

Reply:
Intel likes to hide its software... Installing from source, Spack installations contain a `lib` link that likes to disappear when creating a binary cache. This should be more robust, as it directly references the target of said link.

Reply:
Please consider documenting the reason.

Reply:
Sure, done!